MantidPlot/src/ProjectRecovery.cpp

```diff
@@ -794,6 +799,12 @@ void ProjectRecovery::repairCheckpointDirectory() {
                 "mantid has been unable to successfully handle repair so "
                 "checkpoints may be invalid");
   }
+
+  if (vectorToDelete.size() > 0) {
+    g_log.information("Project Recovery: A repair of the checkpoint directory "
```

Comment:
This is probably the wrong place to log this, a "repair" would only be complete after the broken directories have been removed.

Reply:
That is true :+1: